test/tests/parallel/set1/spam_virus/spam_junk_folder.bats

```diff
@@ -62,12 +62,16 @@ function teardown() { _default_teardown ; }
   )
   _init_with_defaults
   _common_container_setup 'CUSTOM_SETUP_ARGUMENTS'
+  _run_in_container sed -i -e 's/mailbox Junk/mailbox Spam/' /etc/dovecot/conf.d/15-mailboxes.conf
+  assert_success
+  _run_in_container dovecot reload
+  assert_success
 
   _should_send_spam_message
   _should_be_received_by_amavis 'Passed SPAM {RelayedTaggedInbound,Quarantined}'
 
   # Should move delivered spam to Junk folder
```

Comment:
A little more context since we're testing more than one configuration now:

```suggestion
  # Should move delivered spam to the Junk mailbox (adjusted to be located at '.Spam/')
```